code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -390,7 +390,7 @@ GLOBAL_LIST_INIT(durathread_recipes, list ( \
 	force = 0
 	throwforce = 0
 	merge_type = /obj/item/stack/sheet/cotton
-	var/pull_effort = 30
+	var/pull_effort = 10
```

Comment:
This is a part of project make durathread not dumb I assume? Can you elaborate on what got changed?

Reply:
You make it faster and it automatically queues up the next one if it can.